response.go

```diff
@@ -170,7 +170,7 @@ func (s StreamingResponse) Handler() func(*Context) {
 
 // RedirectResponse represents a response that sends a redirect to the client.
 type RedirectResponse struct {
-	// StatusCode provides the 3xx status code of the redirect response.
+	// StatusCode provides the 3xx status code of the redirect response. Default: 308.
```

Comment:
301 or 308. check please

Reply:
we dont know which one the user wants so it doesn't really matter.